src/qp/ensemble.py

```diff
@@ -428,6 +428,21 @@ def sf(self, q):
         """
         return self._frozen.sf(q)
 
+    def logsf(self, q):
```

Comment:
This method was added to support the Anderson-Darling test. It follows the same pattern as all of the other methods inherited from scipy.stats.rv_continuous, such as cdf, sf, etc.